tools/Config.mk

```diff
@@ -48,7 +48,7 @@ else ifeq ($(V),)
 endif
 
 ifeq ($(ECHO_BEGIN),)
-  export ECHO_BEGIN=@echo 
```

Comment:
why change this line

Reply:
I think it just remove extra space that really does not matter for variable assignment

Reply:
This space is useful in order to avoid string concatenation with echo content ...

Reply:
Link issue here: https://github.com/apache/nuttx/pull/8425